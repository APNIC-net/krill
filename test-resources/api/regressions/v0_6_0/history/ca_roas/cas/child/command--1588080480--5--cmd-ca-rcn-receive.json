{
  "actor": "krill",
  "time": "2020-04-28T13:28:00.316544Z",
  "handle": "child",
  "version": 6,
  "sequence": 5,
  "details": {
    "update_rcvd_cert": [
      "0",
      {
        "asn": "",
        "v4": "10.0.0.0/16",
        "v6": "2001:db8::/32"
      }
    ]
  },
  "effect": {
    "events": [
      6
    ]
  }
}