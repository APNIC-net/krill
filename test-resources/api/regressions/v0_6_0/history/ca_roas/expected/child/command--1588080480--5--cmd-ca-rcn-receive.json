{
  "command": {
    "actor": "krill",
    "time": "2020-04-28T13:28:00.316544Z",
    "handle": "child",
    "version": 6,
    "sequence": 5,
    "details": {
      "update_rcvd_cert": [
        "0",
        {
          "asn": "",
          "v4": "10.0.0.0/16",
          "v6": "2001:db8::/32"
        }
      ]
    },
    "effect": {
      "events": [
        6
      ]
    }
  },
  "result": {
    "Events": [
      {
        "id": "child",
        "version": 6,
        "details": {
          "key_pending_to_active": [
            "0",
            {
              "key_id": "77AA69E9571D1DD092F7331CF7157D07B5E8997B",
              "incoming_cert": {
                "cert": "MIIHETCCBfmgAwIBAgIULHbyCriLFMUL3f7w/VSvoDONTdcwDQYJKoZIhvcNAQELBQAwggItMYICKTCCAiUGA1UEAxOCAhwzMDgyMDEwQTAyODIwMTAxMDBENTcwNTJGM0Y3RTI5OTFFNDIyMUQ1NzREMDg5NDc2QjUyMjQxOTJDRjY3N0M5MzkyMjEzMTRFNUU3QTdBNDhBNDFDRkY4QkFBNUU1ODRGMTlEMEU4MkVDRkQ5RDI5Qjg5MTk3MEZCQzRBNTNDREY4NkY2NjVBQURCODM0RDNCQTVCMzU3RTc4RTc3RUQzNTA3QkJFNUY5NUYxNUQ5NzZBNDcwRjdGNzI1NjBEM0EyNTBFM0YwRTU3QzQ5RkMzNEE4MjI2OUREQjYxNDE0NUM5MjBGNEFCRTI2MjM3RTMzMDYyNDYxMTNGNDBFNDc2NTc2QTQ3QzcxNDYxOUU3RUM2ODQzQkM3NkY3RUVBOThGNjY4MzJFQUYwQzE4ODgwREQ1NzQ5RUFFMTMzRTlENzFDNjYxRUM0QkEyNUNCMTk2MDY1Q0IwNDlFRTMyQzFFMjhEOUIwNDE3OEQwMkUzNUFGNTQ0NEQ4RjVDRDU0QzUyMjNENURENjg4MDk3ODFDMDE2NERDMjYzMkRENjc5ODYxNUE2QTFFNEUyRjZFMjI1MDM1M0ExMjMyNzI2Q0FFQzE3MTgzODEyN0U1REM1OEI0NUMyOUI3MUM1MEYxMDU1MjE2NTBFNTkzMjIwQTU2QTkwQjczNEJDODA4MjVDOURDRjIzQUVFNUUxNzcxRDJFOTAyMDMwMTAwMDEwHhcNMjAwNDI4MTMyMzAwWhcNMjEwNDI4MTMyODAwWjAzMTEwLwYDVQQDEyg3N0FBNjlFOTU3MUQxREQwOTJGNzMzMUNGNzE1N0QwN0I1RTg5OTdCMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAxfzcrut62t49w60FaZK5SH4hwuRpgfe/PEPrQiJQ6J+812f+ZL3hxchyMwj5rcfAU0tMVrwKBdeZcfLhM7WsUZ4z11edOK4bCDkdLRUwCOdFp4emOCnQz+OFQNB7gRmGy+vmAe4z2E6MsGKI5JmqFF5g5RfYUPcLOsC0vuq9Kos2/bKuYl7YTUkJmIvxOEkbhiNpTD8yo4u/W+qWLZKRsZgvgTJ49QXYPsD7lXSCBmhypwfpEW6+2eIm5sqaN20RcIknW9ZISrT0KHi2oNdKrJIIGMtjyoeU4BrEcsNvDT9yLxM/Ztlhi7Hf6SL0ftN53SMHiv2LdsdpsArH8jSm7QIDAQABo4ICHzCCAhswDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUd6pp6VcdHdCS9zMc9xV9B7XomXswHwYDVR0jBBgwFoAUbKywsyXpTjVWMCm+YXmGJbjPRHswDgYDVR0PAQH/BAQDAgEGMFkGA1UdHwRSMFAwTqBMoEqGSHJzeW5jOi8vbG9jYWxob3N0L3JlcG8vdGEvMC82Q0FDQjBCMzI1RTk0RTM1NTYzMDI5QkU2MTc5ODYyNUI4Q0Y0NDdCLmNybDA8BggrBgEFBQcBAQQwMC4wLAYIKwYBBQUHMAKGIHJzeW5jOi8vbG9jYWxob3N0L3JlcG8vdGEvdGEuY2VyMIHVBggrBgEFBQcBCwSByDCBxTArBggrBgEFBQcwBYYfcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzBXBggrBgEFBQcwCoZLcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IubWZ0MD0GCCsGAQUFBzANhjFodHRwczovL2xvY2FsaG9zdDozMDAwL3Rlc3QtcnJkcC9ub3RpZmljYXRpb24ueG1sMBgGA1UdIAEB/wQOMAwwCgYIKwYBBQUHDgIwLQYIKwYBBQUHAQcBAf8EHjAcMAsEAgABMAUDAwAKADANBAIAAjAHAwUAIAENuDANBgkqhkiG9w0BAQsFAAOCAQEAr3X0I58GF+kEeLCQCle/RhUojMMra86p0JPV9bMYEoc56oRTMxlQBc+cx8v6NRaHJeigObOZCg9MQL5Wh0sFl1u7Y1lknGLro03eJ9wyOP1k4h74VSJ2YP2rLaV6Q10YK/hRs86agdAVxr3WL3kFG8802VUBqF75JlVTHZ2WpsuaQ2BG2f8JBZUrghDGt+0/Sjit8UrANuECmHtEF5MGSs0nbgH7bn/vviRLUdWZKpxGsZ3SqgW2G/iGGVK6eJqXRjbtXEDwATIdodDVodEK/2yCwv6rEzIVtdw+b44GFNcOcHyalLGKR8HDu2RqiRcVsOjshuRwqJ/FdrDW8ifjAg==",
                "uri": "rsync://localhost/repo/ta/0/77AA69E9571D1DD092F7331CF7157D07B5E8997B.cer",
                "resources": {
                  "asn": "",
                  "v4": "10.0.0.0/16",
                  "v6": "2001:db8::/32"
                }
              },
              "current_set": {
                "number": 1,
                "revocations": [],
                "manifest_info": {
                  "name": "77AA69E9571D1DD092F7331CF7157D07B5E8997B.mft",
                  "current": {
                    "content": "MIIJRAYJKoZIhvcNAQcCoIIJNTCCCTECAQMxDzANBglghkgBZQMEAgEFADCBmwYLKoZIhvcNAQkQARqggYsEgYgwgYUCAQEYDzIwMjAwNDI4MTMyMzAwWhgPMjAyMDA0MjkxMzI4MDBaBglghkgBZQMEAgEwUzBRFiw3N0FBNjlFOTU3MUQxREQwOTJGNzMzMUNGNzE1N0QwN0I1RTg5OTdCLmNybAMhADOwUJTKgfvzUJqIqIHR2VdBtuHYmYoWCgNUdi/Mpf9WoIIGyzCCBscwggWvoAMCAQICFF6KnUdCe+tzp1idD2SbiMhmmivzMA0GCSqGSIb3DQEBCwUAMDMxMTAvBgNVBAMTKDc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IwHhcNMjAwNDI4MTMyMzAwWhcNMjAwNTA1MTMyODAwWjCCAi0xggIpMIICJQYDVQQDE4ICHDMwODIwMTBBMDI4MjAxMDEwMEFCRTFEN0Q0NEE0RTgyNzU2NkQ2RDU4RjQyNkMzMDQ0NzA3RjI4MENCNDc3QjJGNzMyNkJGRTA2QzM5MjMxNEFDRTc0MEVEOUMwQTVDMEMyMzUxNTQ3REFGOUM4MzhCQTJDRUE3NkQ2QTVCMTM4QTE1NEEyQjkwQkJCMThCOTJDMEZGOTkyNTRBQUUyRUFFNTZDNzFFMTczQTY3N0UyRUNDRDE3OEI5MzI5RTdBNkFCRDQ4NTY5QUFDMzEwRjI4Rjg3MTkzOEQ0OUM4MkZCNDBCMTFFODZFNjc3QzczMjFDMkMxNERGQjkwRkI3MDgxNzZCMkNFM0VGQTJFRTE0NEJCODFCQTFGODY0QjNDODJFNjE2QzgwQ0M3OERBQ0Y2RURFMTIzRjE4QTcwRUJGRjAxNTY0QTlDQTQ2RjQ5RjU2NzI3QjUwNTkwRjg3OTYxNENFQTFGMkU5NzY0MzE4MTUzODZDMTI3OENDM0Y1NjAxOTY2RTBDMjA2NzUxMzUxRDU3NkU0QzRGNzQwNTBGODY4OTUwMTlDNEUxRDYzOTZBM0IzNDdFMDUyNThBQTkyMjc3OEZEQTczNEFENzZDQTY0MEU2RUNENjMwODhCNkUyREEzRjg1M0U2RkY4QzA1NTk1QTkyMjgyMDdCNDAxNTU1NURFMzQxQkUzQTA4QzhEMDIwMzAxMDAwMTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAKvh19RKToJ1ZtbVj0JsMERwfygMtHey9zJr/gbDkjFKznQO2cClwMI1FUfa+cg4uizqdtalsTihVKK5C7sYuSwP+ZJUquLq5Wxx4XOmd+LszReLkynnpqvUhWmqwxDyj4cZONScgvtAsR6G5nfHMhwsFN+5D7cIF2ss4++i7hRLuBuh+GSzyC5hbIDMeNrPbt4SPxinDr/wFWSpykb0n1Zye1BZD4eWFM6h8ul2QxgVOGwSeMw/VgGWbgwgZ1E1HVduTE90BQ+GiVAZxOHWOWo7NH4FJYqpIneP2nNK12ymQObs1jCItuLaP4U+b/jAVZWpIoIHtAFVVd40G+OgjI0CAwEAAaOCAdUwggHRMB0GA1UdDgQWBBSROgQFi4tSiMORBSGZefQr7ZXfOjAfBgNVHSMEGDAWgBR3qmnpVx0d0JL3Mxz3FX0HteiZezAOBgNVHQ8BAf8EBAMCB4AwXAYDVR0fBFUwUzBRoE+gTYZLcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IuY3JsMGQGCCsGAQUFBwEBBFgwVjBUBggrBgEFBQcwAoZIcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby90YS8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IuY2VyMGcGCCsGAQUFBwELBFswWTBXBggrBgEFBQcwC4ZLcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IubWZ0MBgGA1UdIAEB/wQOMAwwCgYIKwYBBQUHDgIwIQYIKwYBBQUHAQcBAf8EEjAQMAYEAgABBQAwBgQCAAIFADAVBggrBgEFBQcBCAEB/wQGMASgAgUAMA0GCSqGSIb3DQEBCwUAA4IBAQAgpq38Iqek9M/9QxRia8QalyAGRHNDYFex7ivpFQf9bF8UVx4aCNEUdGDGcGvlzlV6MG6Y/H/ixyk8YoBn/zejms2C+GtRjFweTiaVLTsK8jk7NPQKHgDVunFb6gir8PUMeRf8w+G0g80T4T45ycsSwVaUJbRJNwg1ftiMuO6c0bw72AzcUK5PQamN+Go2K1eUdpB6zkRoday+DGSe2UGKaiYMgbQct2fpgOH0dFp4WUmbizH3bUDWkHCJr8oAHdUhFPUyP4ao3uZbvzU3xY18tiD/CDSEnBnwv1BHkGOXPNvTxHmlLkV1omgU7i/QBL+5xvPTrPWIz1aTg2COwv89MYIBrDCCAagCAQOAFJE6BAWLi1KIw5EFIZl59Cvtld86MA0GCWCGSAFlAwQCAQUAoGswGgYJKoZIhvcNAQkDMQ0GCyqGSIb3DQEJEAEaMBwGCSqGSIb3DQEJBTEPFw0yMDA0MjgxMzI4MDBaMC8GCSqGSIb3DQEJBDEiBCCgjHS8yb381FSdNhUl4EV+ItFCnV4LFNErb9g7hrhfQTANBgkqhkiG9w0BAQEFAASCAQCCZpfQPdNapE2KFGYu4EvJcbKP4s8Zm/dHBsLuEBd8Q4T6Jcjksin2r6TTE0QQ0iLLx+aPBpokQvur7MT9w7Msgz0pA0J/NHrxmFaU4CveIs5XirMVMPy/xB0rgS/tVMTZE6nn5U9n1o7VXJA7m4pkrclsBMrmGtE8B2i2JJzIK7fhCxry/K3HrU5+T65pnw9Pv8ALPp7vVGOZiRY+vZFEuCBGLYWsL8DEsaYYQF9SRh+fsY7KoHgGaS4pcN3ricFybgfo5Gptic3WwLKV/d6JJjP0jtucVyhu83ds6UELJRSJ3UrlMmgmNngmAmFGdqmeNVkVnmU2RRdKSaObBbyC",
                    "serial": "539736336172149954781492781987142862243672763379",
                    "expires": "2020-04-29T13:28:00.320089Z"
                  },
                  "next_update": "2020-04-29T13:28:00.320089Z",
                  "old": null
                },
                "crl_info": {
                  "name": "77AA69E9571D1DD092F7331CF7157D07B5E8997B.crl",
                  "current": {
                    "content": "MIIBrzCBmAIBATANBgkqhkiG9w0BAQsFADAzMTEwLwYDVQQDEyg3N0FBNjlFOTU3MUQxREQwOTJGNzMzMUNGNzE1N0QwN0I1RTg5OTdCFw0yMDA0MjgxMzIzMDBaFw0yMDA0MjkxMzI4MDBaMACgLzAtMB8GA1UdIwQYMBaAFHeqaelXHR3QkvczHPcVfQe16Jl7MAoGA1UdFAQDAgEBMA0GCSqGSIb3DQEBCwUAA4IBAQArK9C7Ibyho/XO2dcAXEwy+o+aOW74cAdeYKMhFeA0SBK0I774Fh/dAlq8TIvra+KSpOf/917utLur3qveR8N50O+y4aaFGYNVc6iAp0Q5LsmjAlibaqzwteneDspGHwKU2MltOkUkP8bbCMSPooZ9TLZpGrk1lZs9lR4dbl46efIK+ETeU/eXJULciPUJYcUVGGtVFCxfBZUcx1NmWHP9YFsa3lokS6LMofqnBsGbfqmxkTJPC72I2SLk1aF7DRcFU/50cnZGc11MHTRzORbp/2aGriEA2t1aZjdke9q8SiJUWGm2vcHJjnE+fw0QDi8G5ZVh1vBe0OtQW0t5SHQR",
                    "serial": "1",
                    "expires": "2020-04-29T13:28:00.316731Z"
                  },
                  "old": null
                }
              },
              "request": null
            },
            {
              "ca_repo": "rsync://localhost/repo/child/0/",
              "added": [
                {
                  "name": "77AA69E9571D1DD092F7331CF7157D07B5E8997B.mft",
                  "object": {
                    "content": "MIIJRAYJKoZIhvcNAQcCoIIJNTCCCTECAQMxDzANBglghkgBZQMEAgEFADCBmwYLKoZIhvcNAQkQARqggYsEgYgwgYUCAQEYDzIwMjAwNDI4MTMyMzAwWhgPMjAyMDA0MjkxMzI4MDBaBglghkgBZQMEAgEwUzBRFiw3N0FBNjlFOTU3MUQxREQwOTJGNzMzMUNGNzE1N0QwN0I1RTg5OTdCLmNybAMhADOwUJTKgfvzUJqIqIHR2VdBtuHYmYoWCgNUdi/Mpf9WoIIGyzCCBscwggWvoAMCAQICFF6KnUdCe+tzp1idD2SbiMhmmivzMA0GCSqGSIb3DQEBCwUAMDMxMTAvBgNVBAMTKDc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IwHhcNMjAwNDI4MTMyMzAwWhcNMjAwNTA1MTMyODAwWjCCAi0xggIpMIICJQYDVQQDE4ICHDMwODIwMTBBMDI4MjAxMDEwMEFCRTFEN0Q0NEE0RTgyNzU2NkQ2RDU4RjQyNkMzMDQ0NzA3RjI4MENCNDc3QjJGNzMyNkJGRTA2QzM5MjMxNEFDRTc0MEVEOUMwQTVDMEMyMzUxNTQ3REFGOUM4MzhCQTJDRUE3NkQ2QTVCMTM4QTE1NEEyQjkwQkJCMThCOTJDMEZGOTkyNTRBQUUyRUFFNTZDNzFFMTczQTY3N0UyRUNDRDE3OEI5MzI5RTdBNkFCRDQ4NTY5QUFDMzEwRjI4Rjg3MTkzOEQ0OUM4MkZCNDBCMTFFODZFNjc3QzczMjFDMkMxNERGQjkwRkI3MDgxNzZCMkNFM0VGQTJFRTE0NEJCODFCQTFGODY0QjNDODJFNjE2QzgwQ0M3OERBQ0Y2RURFMTIzRjE4QTcwRUJGRjAxNTY0QTlDQTQ2RjQ5RjU2NzI3QjUwNTkwRjg3OTYxNENFQTFGMkU5NzY0MzE4MTUzODZDMTI3OENDM0Y1NjAxOTY2RTBDMjA2NzUxMzUxRDU3NkU0QzRGNzQwNTBGODY4OTUwMTlDNEUxRDYzOTZBM0IzNDdFMDUyNThBQTkyMjc3OEZEQTczNEFENzZDQTY0MEU2RUNENjMwODhCNkUyREEzRjg1M0U2RkY4QzA1NTk1QTkyMjgyMDdCNDAxNTU1NURFMzQxQkUzQTA4QzhEMDIwMzAxMDAwMTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAKvh19RKToJ1ZtbVj0JsMERwfygMtHey9zJr/gbDkjFKznQO2cClwMI1FUfa+cg4uizqdtalsTihVKK5C7sYuSwP+ZJUquLq5Wxx4XOmd+LszReLkynnpqvUhWmqwxDyj4cZONScgvtAsR6G5nfHMhwsFN+5D7cIF2ss4++i7hRLuBuh+GSzyC5hbIDMeNrPbt4SPxinDr/wFWSpykb0n1Zye1BZD4eWFM6h8ul2QxgVOGwSeMw/VgGWbgwgZ1E1HVduTE90BQ+GiVAZxOHWOWo7NH4FJYqpIneP2nNK12ymQObs1jCItuLaP4U+b/jAVZWpIoIHtAFVVd40G+OgjI0CAwEAAaOCAdUwggHRMB0GA1UdDgQWBBSROgQFi4tSiMORBSGZefQr7ZXfOjAfBgNVHSMEGDAWgBR3qmnpVx0d0JL3Mxz3FX0HteiZezAOBgNVHQ8BAf8EBAMCB4AwXAYDVR0fBFUwUzBRoE+gTYZLcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IuY3JsMGQGCCsGAQUFBwEBBFgwVjBUBggrBgEFBQcwAoZIcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby90YS8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IuY2VyMGcGCCsGAQUFBwELBFswWTBXBggrBgEFBQcwC4ZLcnN5bmM6Ly9sb2NhbGhvc3QvcmVwby9jaGlsZC8wLzc3QUE2OUU5NTcxRDFERDA5MkY3MzMxQ0Y3MTU3RDA3QjVFODk5N0IubWZ0MBgGA1UdIAEB/wQOMAwwCgYIKwYBBQUHDgIwIQYIKwYBBQUHAQcBAf8EEjAQMAYEAgABBQAwBgQCAAIFADAVBggrBgEFBQcBCAEB/wQGMASgAgUAMA0GCSqGSIb3DQEBCwUAA4IBAQAgpq38Iqek9M/9QxRia8QalyAGRHNDYFex7ivpFQf9bF8UVx4aCNEUdGDGcGvlzlV6MG6Y/H/ixyk8YoBn/zejms2C+GtRjFweTiaVLTsK8jk7NPQKHgDVunFb6gir8PUMeRf8w+G0g80T4T45ycsSwVaUJbRJNwg1ftiMuO6c0bw72AzcUK5PQamN+Go2K1eUdpB6zkRoday+DGSe2UGKaiYMgbQct2fpgOH0dFp4WUmbizH3bUDWkHCJr8oAHdUhFPUyP4ao3uZbvzU3xY18tiD/CDSEnBnwv1BHkGOXPNvTxHmlLkV1omgU7i/QBL+5xvPTrPWIz1aTg2COwv89MYIBrDCCAagCAQOAFJE6BAWLi1KIw5EFIZl59Cvtld86MA0GCWCGSAFlAwQCAQUAoGswGgYJKoZIhvcNAQkDMQ0GCyqGSIb3DQEJEAEaMBwGCSqGSIb3DQEJBTEPFw0yMDA0MjgxMzI4MDBaMC8GCSqGSIb3DQEJBDEiBCCgjHS8yb381FSdNhUl4EV+ItFCnV4LFNErb9g7hrhfQTANBgkqhkiG9w0BAQEFAASCAQCCZpfQPdNapE2KFGYu4EvJcbKP4s8Zm/dHBsLuEBd8Q4T6Jcjksin2r6TTE0QQ0iLLx+aPBpokQvur7MT9w7Msgz0pA0J/NHrxmFaU4CveIs5XirMVMPy/xB0rgS/tVMTZE6nn5U9n1o7VXJA7m4pkrclsBMrmGtE8B2i2JJzIK7fhCxry/K3HrU5+T65pnw9Pv8ALPp7vVGOZiRY+vZFEuCBGLYWsL8DEsaYYQF9SRh+fsY7KoHgGaS4pcN3ricFybgfo5Gptic3WwLKV/d6JJjP0jtucVyhu83ds6UELJRSJ3UrlMmgmNngmAmFGdqmeNVkVnmU2RRdKSaObBbyC",
                    "serial": "539736336172149954781492781987142862243672763379",
                    "expires": "2020-04-29T13:28:00.320089Z"
                  }
                },
                {
                  "name": "77AA69E9571D1DD092F7331CF7157D07B5E8997B.crl",
                  "object": {
                    "content": "MIIBrzCBmAIBATANBgkqhkiG9w0BAQsFADAzMTEwLwYDVQQDEyg3N0FBNjlFOTU3MUQxREQwOTJGNzMzMUNGNzE1N0QwN0I1RTg5OTdCFw0yMDA0MjgxMzIzMDBaFw0yMDA0MjkxMzI4MDBaMACgLzAtMB8GA1UdIwQYMBaAFHeqaelXHR3QkvczHPcVfQe16Jl7MAoGA1UdFAQDAgEBMA0GCSqGSIb3DQEBCwUAA4IBAQArK9C7Ibyho/XO2dcAXEwy+o+aOW74cAdeYKMhFeA0SBK0I774Fh/dAlq8TIvra+KSpOf/917utLur3qveR8N50O+y4aaFGYNVc6iAp0Q5LsmjAlibaqzwteneDspGHwKU2MltOkUkP8bbCMSPooZ9TLZpGrk1lZs9lR4dbl46efIK+ETeU/eXJULciPUJYcUVGGtVFCxfBZUcx1NmWHP9YFsa3lokS6LMofqnBsGbfqmxkTJPC72I2SLk1aF7DRcFU/50cnZGc11MHTRzORbp/2aGriEA2t1aZjdke9q8SiJUWGm2vcHJjnE+fw0QDi8G5ZVh1vBe0OtQW0t5SHQR",
                    "serial": "1",
                    "expires": "2020-04-29T13:28:00.316731Z"
                  }
                }
              ],
              "updated": [],
              "withdrawn": []
            }
          ]
        }
      }
    ]
  }
}